src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -297,13 +420,13 @@ const HoloFuelDnaInterface = {
     acceptMany: async (transactionIdArray) => {
       const result = await createZomeCall('transactions/receive_payments_pending')({ promises: transactionIdArray })
       const transactionArray = Object.entries(result).map(presentAcceptedPayment)
-      return transactionArray.sort((a, b) => a.timestamp < b.timestamp ? -1 : 1)
+      return transactionArray.sort((a, b) => a.timestamp < b.timestamp ? -1 : 1).reverse()
```

Comment:
rather than calling reverse here, you can just change the logic in the `sort` function. Either switching the `-1` and `1` or replacing the `<` with `>` will do it.

Reply:
Hmm... Strange!! ...I actually updated this to exactly what you stated here, but it seems that was overwritten in the merge from 1170 to 4770.  Anyway, I agree it's preferable. :100: :1st_place_medal: